docs/fxa-engineering/fxa-logging.md

```diff
@@ -0,0 +1,54 @@
+---
+id: fxa-logging
+title: Application Logging
+sidebar_label: Application Logging
+---
+
+## Application Logging
+
+In FxA Javascript/Typescript server-side applications, logging is emitted via [mozlog] which adheres to the [mozlog schema] format. The log messages are captured from `stdout` and aggregated into Google [BigQuery]. Application exceptions are captured via [Sentry] and sent to our Sentry instance.
+
+Stage and development logging is grouped under the `fxa-nonprod` project, while accessing the `fxa-prod` logs requires additional special access privileges.
+
+### Capturing Event Details
+
+Once a `mozlog` instance is created, log events are captured with a dot concatenated string indicating an (ideally) unique location of the logging statement that is clear where in the code-base it was made, along with any additional fields to include.
```

Comment:
:+1: Looks good. Might be helpful to add an example of creating a `mozlog` instance, since a config object is generally passed in at initialization time, for instance, `require('mozlog')(config.logging)`, where `config.logging` looks like https://github.com/mozilla/fxa/blob/9a6c1425669fa5818b3896faf4777c14d15c71d8/packages/fxa-content-server/server/lib/configuration.js#L347-L357.

Reply:
Cool, added!